data/themes/darktable.css

```diff
@@ -1157,16 +1157,51 @@ notebook tab:hover label,
   color: @button_hover_fg;
 }
 
+#history-button *
+{
+  color: @plugin_fg_color;
+  background-color: transparent;
+}
+
+#history-button-always-enabled *,
+#history-button-default-enabled *
+{
+  color: @disabled_fg_color;
```

Comment:
Fine with me.

Reply:
@TurboGit @Nilvus @jenshannoschwalm Any thoughts about a single button placed beside original in the stack that would collapse or open these modules in the stack. Then the stack would start visually with original and could be set to reset to that on a full compress to original and the other modules could be displayed by clicking the single button beside original...then in the future modules could be added to this group and not cause any physical issue for users ...just a thought and not sure if its too late but I think this would work and also no need to come up with a scheme that works visually for everyone...does that make sense....This could also be documented that these modules are applied automatically ie not really the history and the history would really now be the history of the applied user commands?? Just a thought

Reply:
Collapsing rather than de-contrasting sounds like an interesting idea to me... it would be nice to free up the space. Not sure if that pushes this into "new feature" territory, though...